app/src/main/res/layout/settings_activity.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <androidx.fragment.app.FragmentContainerView
+        android:id="@+id/settings_container"
```

Comment:
Please name it `settingsContainer`. See the code style here: https://github.com/AntennaPod/AntennaPod/wiki/Code-style

Reply:
Oh I forgot that you do camelCase, lol. I'll fix it.